src/ExpressiveInstaller/Resources/config/routes-full.php

```diff
@@ -24,6 +24,8 @@
  *     Zend\Expressive\Router\Route::HTTP_METHOD_ANY,
  *     'contact'
  * );
+ *
+ * @var \Zend\Expressive\Application $app
```

Comment:
Somehow this feels weird. That docblock is a page level docblock.  Wouldn't it be better to write in a separate line like this: 

` /** @var \Zend\Expressive\Application $app */`